apps/web/src/components/Basenames/ManageNames/NamesList.tsx

```diff
@@ -86,6 +97,43 @@ export default function NamesList() {
           />
         ))}
       </ul>
+
+      {/* Pagination Controls */}
+      {(hasPrevious || hasNext || currentPageNumber > 1) && (
```

Comment:
I think checking `currentPageNumber` is redundant. Would there ever be a case where `hasPrevious` and `hasNext` are `false` but the current page number is > 1?